### PR TITLE
Fix Jupyter Notebook $HOME Permission

### PR DESCRIPTION
**Pull request by Tabrizian**
About #4863

But the permission of $HOME was not being affected in the script not sure whether this PR solves the problem.

/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/4867)

### CODE REVIEW

File: components/tensorflow-notebook-image/Dockerfile [hunk 1]
```diff
@@ -86,7 +86,6 @@ ENV LANGUAGE en_US.UTF-8
 # but allow for non-initial launches of the notebook to have
 # $HOME provided by the contents of a PV
 RUN useradd -M -s /bin/bash -N -u $NB_UID $NB_USER && \
-    chown -R ${NB_USER}:users /usr/local/bin && \
```
Comment (jlewi) [inline components/tensorflow-notebook-image/Dockerfile]: Why did you remove that line?
Don't we just want to add

```
chown -R ${NB_USER}:users ${HOME}
```

And then you could verify by  building the docker image and doing ls -la /home
hopefully /home/jovyan should be owned by user jovyan and group users.

Comment (Tabrizian) [reply components/tensorflow-notebook-image/Dockerfile]: Sorry, I was really stupid. Fixed!

**Comment by jlewi** [comment]
Thanks!
/lgtm
/approve

Not sure about the tests though.

**Comment by jlewi** [comment]
Its the TF notebook release that's failing.
http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-presubmit-tf-notebook-release-4867-2a0406e-6592-abaf?tab=workflow

It looks like building the jupyter images is failing. So could be an issue in the Dockerfile

**Comment by Tabrizian** [comment]
@jlewi This needs a LGTM only
